docs/02.API-REFERENCE.md

```diff
@@ -584,6 +584,32 @@ typedef void (*jerry_object_native_free_callback_t) (void *native_p);
 - [jerry_object_native_info_t](#jerry_object_native_info_t)
 - [jerry_create_arraybuffer_external](#jerry_create_arraybuffer_external)
 
+## jerry_error_object_update_callback_t
+
+**Summary**
+
+Decorator callback for Error objects. This native callback is called every time when an Error
+object is created and the decorator can create or update any properties of the newly created
+Error object.
+
+*Note*:
+ - The callback function cannot be called recursively, so the Error objects created
+   when the callback is running are not updated.
+
+*New in version [[NEXT_RELEASE]]*.
+
+**Prototype**
+
+```c
+typedef void (*jerry_error_object_update_callback_t) (const jerry_value_t error_object);
```

Comment:
To keep it consistent with vm_stop callback, shouldn't a `void* user_p` need to be passed to this callback?

Reply:
But that means extra data. Do we need it?